client/web/src/enterprise/batches/MultiSelectContext.tsx

```diff
@@ -42,13 +46,16 @@ const defaultState = (): MultiSelectContextState => ({
     visible: new Set(),
     areAllVisibleSelected: () => false,
     isSelected: () => false,
-    deselectAll: () => {},
-    deselectVisible: () => {},
-    deselectSingle: () => {},
-    selectAll: () => {},
-    selectVisible: () => {},
-    selectSingle: () => {},
-    setVisible: () => {},
+    deselectAll: noop,
+    deselectVisible: noop,
+    deselectSingle: noop,
+    selectAll: noop,
+    selectVisible: noop,
+    toggleAll: noop,
+    toggleVisible: noop,
+    toggleSingle: noop,
+    selectSingle: noop,
+    setVisible: noop,
```

Comment:
This is the most ridiculous request in the world, but can we make this alphabetical, please?

```suggestion
    selectAll: noop,
    selectSingle: noop,
    selectVisible: noop,
    toggleAll: noop,
    toggleSingle: noop,
    toggleVisible: noop,

    setVisible: noop,
```

Reply:
Hahaha I LOVE alphabetizing object keys actually, I just didn't want to here since you initially set the ordering! It would be my pleasure. 😂